source/overview/license-and-subscription.rst

```diff
@@ -0,0 +1,126 @@
+
+Applying the license key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   If you are not a System Admin, you will not have access to the System Console to apply the license.
+
+Navigate to the **Subscriptions** page of your Mattermost account. Select **Download License** to download the license key for your subscription.
+
+You can apply the license in the Mattermost System Console: **System Console > About > Edition and License**. Follow the steps provided to apply your license key.
+You can also use the `CLI <https://docs.mattermost.com/install/ee-install.html#changing-a-license-key>`__ to apply the license.
+
+Viewing license information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can view your license start date, expiry date, number of users and license type in your account under **Subscription**.
+
+Adding more users to a purchased license
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can add more users during your subscription period without requesting a license.
+
+During the annual renewal, a retroactive charge will be placed for any unique users added during the past subscription period that is
+above the licensed total unique users in the current paid subscription. The retroactive charge per user will be the initial subscription
+cost per user.
+
+Renewing an Enterprise license
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ability to renew your license in the Customer Portal will be available in a future release.
+To renew your license in the meantime, contact your CSM to start the renewal process.
+
+Frequently Asked Questions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Are my credit card details safe?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We use `Stripe <https://stripe.com/payments>`_ to process credit card transactions. We do not store any credit card details at any stage. Payments made by credit card are authenticated using `3D Secure <https://support.payfast.co.za/article/96-what-is-3d-secure-visa-secure-mastercard-securecode>`__ which is PCI-DSS compliant.
+
+Should you wish to make payment using another method, please contact our Billing team.
+
+
+How is user defined for Enterprise Edition subscriptions?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+See our `frequently asked questions about licensing <https://about.mattermost.com/pricing/#faq>`__.
+
+
+Do I need to pay for deactivated users?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No. If you deactivate a user that user is not counted as an active user during your annual renewal process. You can deactivate users manually via System Console and also via Active Directory/LDAP synchronization, the CLI tool, and the server APIs.
+
+If you choose to pull SQL reports from the database to monitor individual activity to make deactivation decisions, and you are running under high user load, we recommend the reports are pulled from a read replica of the database.
+
+Can I use the same license key on multiple Enterprise Edition servers?
```

Comment:
Elaine, Tim, and Rachel brought up issues with this question. This question and answer are already in the docs as written (moved from another file). I'm going to look into changing the content here.

Reply:
Let me know whether it'll be best to just remove it for now.

Reply:
@TQuock, I've updated the wording. LMK if it works better.